frontend/src/static/js/v_authorship.js

```diff
@@ -315,10 +323,11 @@ window.vAuthorship = {
     },
   },
 
-  created() {
```

Comment:
One more question, can you justify the change from `created` to `mounted`?

Reply:
My bad, changing it back.